h/realtime.py

```diff
@@ -88,7 +88,7 @@ def publish_user(self, payload):
         self._publish("user", payload)
 
     def _publish(self, routing_key, payload):
-        try:
+        try:  # pylint: disable=too-many-try-statements
```

Comment:
This is a bit nest-tastic but I think it's fine as is / splitting it up would make it worse.